pysat/tests/classes/cls_instrument_property.py

```diff
@@ -45,7 +49,14 @@ def test_basic_instrument_bad_keyword_init(self):
     @pytest.mark.parametrize('kwarg', ['supported_tags', 'start', 'stop',
```

Comment:
This may potentially be updated to use the list of reserved keywords attached to each Instrument object. `inst.kwargs_reserved`

Reply:
So I tried putting in `pysat.Instrument().kwargs_reserved` but the tests fail since that list also includes names, inst_id, and tag.

Reply:
OK.  Recommending we leave as is.